[2023-09-03T10:06:41.355+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: producer.update_dataset_2 scheduled__2023-09-02T00:00:00+00:00 [queued]>
[2023-09-03T10:06:41.381+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: producer.update_dataset_2 scheduled__2023-09-02T00:00:00+00:00 [queued]>
[2023-09-03T10:06:41.385+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-09-03T10:06:41.386+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-09-03T10:06:41.386+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-09-03T10:06:41.404+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): update_dataset_2> on 2023-09-02 00:00:00+00:00
[2023-09-03T10:06:41.414+0000] {standard_task_runner.py:55} INFO - Started process 69 to run task
[2023-09-03T10:06:41.419+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'producer', 'update_dataset_2', 'scheduled__2023-09-02T00:00:00+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/producer.py', '--cfg-path', '/tmp/tmp9nx4m6q_']
[2023-09-03T10:06:41.426+0000] {standard_task_runner.py:83} INFO - Job 94: Subtask update_dataset_2
[2023-09-03T10:06:41.569+0000] {task_command.py:376} INFO - Running <TaskInstance: producer.update_dataset_2 scheduled__2023-09-02T00:00:00+00:00 [running]> on host a538dd8bfd1d
[2023-09-03T10:06:41.661+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=producer
AIRFLOW_CTX_TASK_ID=update_dataset_2
AIRFLOW_CTX_EXECUTION_DATE=2023-09-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-02T00:00:00+00:00
[2023-09-03T10:06:41.663+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-09-03T10:06:41.675+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/baseoperator.py:1390: RemovedInAirflow3Warning: Passing 'execution_date' to 'TaskInstance.xcom_push()' is deprecated.
  context['ti'].xcom_push(key=key, value=value, execution_date=execution_date)

[2023-09-03T10:06:41.740+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=producer, task_id=update_dataset_2, execution_date=20230902T000000, start_date=20230903T100641, end_date=20230903T100641
[2023-09-03T10:06:41.860+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-09-03T10:06:41.883+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-03T15:04:16.761+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: producer.update_dataset_2 scheduled__2023-09-02T00:00:00+00:00 [queued]>
[2023-09-03T15:04:16.773+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: producer.update_dataset_2 scheduled__2023-09-02T00:00:00+00:00 [queued]>
[2023-09-03T15:04:16.773+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-09-03T15:04:16.773+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-09-03T15:04:16.773+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-09-03T15:04:16.781+0000] {taskinstance.py:1383} INFO - Executing <Task(_PythonDecoratedOperator): update_dataset_2> on 2023-09-02 00:00:00+00:00
[2023-09-03T15:04:16.786+0000] {standard_task_runner.py:55} INFO - Started process 482 to run task
[2023-09-03T15:04:16.790+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'producer', 'update_dataset_2', 'scheduled__2023-09-02T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/producer.py', '--cfg-path', '/tmp/tmpy27b6gda']
[2023-09-03T15:04:16.790+0000] {standard_task_runner.py:83} INFO - Job 12: Subtask update_dataset_2
[2023-09-03T15:04:16.855+0000] {task_command.py:376} INFO - Running <TaskInstance: producer.update_dataset_2 scheduled__2023-09-02T00:00:00+00:00 [running]> on host c0ddfbe053d4
[2023-09-03T15:04:16.902+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=producer
AIRFLOW_CTX_TASK_ID=update_dataset_2
AIRFLOW_CTX_EXECUTION_DATE=2023-09-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-02T00:00:00+00:00
[2023-09-03T15:04:16.903+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-09-03T15:04:16.904+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/models/baseoperator.py:1390: RemovedInAirflow3Warning: Passing 'execution_date' to 'TaskInstance.xcom_push()' is deprecated.
  context['ti'].xcom_push(key=key, value=value, execution_date=execution_date)

[2023-09-03T15:04:16.920+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=producer, task_id=update_dataset_2, execution_date=20230902T000000, start_date=20230903T150416, end_date=20230903T150416
[2023-09-03T15:04:16.962+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-09-03T15:04:16.977+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
